client-react/src/pages/app/deployment-center/code/DeploymentCenterCodeForm.tsx

```diff
@@ -720,7 +720,7 @@ const DeploymentCenterCodeForm: React.FC<DeploymentCenterCodeFormProps> = props
             />
           </>
           <div className={pivotContent}>
-            <DeploymentCenterCodePivot {...props} formProps={formProps} />
+            <DeploymentCenterCodePivot {...props} tab={props.tab} formProps={formProps} />
```

Comment:
`tab` should already be set because of the spread before it.

```suggestion
            <DeploymentCenterCodePivot {...props} formProps={formProps} />
```